src/Popup.css

```diff
@@ -8,26 +11,74 @@
     bottom:0;
     margin: auto;
     background-color: rgba(0,0,0,0.5);
+    z-index: 2;
 }
 
 .popup.hidden {
     display:none;
 }
 
-
 .popupInner {
     position:absolute;
     left:35%;
-    right: 35%;
     top: 35%;
-    bottom: 45%;
+    max-width: 250px;
+    min-width: 200px;
+    min-height: 210px;
+    overflow: hidden;
     margin: auto;
     border-radius: 20px;
     background: white;
 }
 
 
+.popupHeader {
+    font-size: 30px;
+}
+
+/* Style for Idea Edit box */
+
+.editHeader {
+    font-size:22px;
+}
+
+.editWrapper {
+    position: fixed;
+    width:100%;
+    height: 100%;
+    top:0;
+    left:0;
+    right:0;
+    bottom:0;
+    margin: auto;
+    background-color: rgba(0,0,0,0.5);
+    z-index: 2;
+}
+
+.edit {
+    position:absolute;
+    left:35%;
+    top: 35%;
```

Comment:
These values feel like you wanted it to be centered and so you futzed with the numbers until it looked about center.

Reply:
Yea pretty much. 
I had tried centering stuff previously, and ended up only figuring out how to push it down and over as I did.
What should be used? maybe it was different context, but previously I had tried "align-self" and "align-content: center" I believe

Reply:
Look up centering things with flexbox.